package.json

```diff
@@ -29,9 +27,8 @@
     "webpack-manifest-plugin": "^1.3.1"
   },
   "devDependencies": {
-    "elm-hot-loader": "^0.5.4",
     "webpack": "^3.12.0",
     "webpack-dev-server": "^2.7.1",
     "webpack-merge": "^4.2.2"
   }
-}
+}
```

Comment:
@NikitaRubocki In the future, avoid removal of trailing newlines at the end of a file. Sometimes this is caused by an editor configuration (in bad taste).

Reply:
Noted, I updated the setting in VSCode